src/deepEqual.js

```diff
@@ -21,6 +21,20 @@
  */
 function deepEqual(a, b) {
   // write code here
+  let isNotObject = arg => typeof arg !== 'object' || arg === null;
```

Comment:
isNotSmth() are bad functions. It's easier to work with isSmth() functions. 

Looks at examples below. Second one is easier to read.
```
if(!isNotSmth(...)) {
  // do something
}
```

```
if(!isSmth(...)) {
  // do something
}
```